app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -79,15 +79,17 @@ function add_stage1() {
 }
 
 src_prepare() {
-	# ensure we use a CoreOS PXE image version that matches rkt's expectations.
-	local rkt_coreos_version
+	eapply_user
```

Comment:
was this previously in `autotools-utils_src_prepare`?

Reply:
Yes, and it's required to call it since EAPI 6.